scipy/stats/stats.py

```diff
@@ -3736,13 +3781,25 @@ def pearsonr(x, y):
         Input array.
     y : (N,) array_like
         Input array.
+    alternative : {'two-sided', 'greater', 'less'}, optional
+        Indicates the alternative hypothesis. The default value is
+        'two-sided'.
```

Comment:
I think we should be more explicit about what the alternatives mean, personally. There